miso-web/src/main/webapp/scripts/list_library.js

```diff
@@ -139,9 +139,10 @@ ListTarget.library = {
         }];
   },
   searchTermSelector: function(searchTerms) {
-    const plainSampleTerms = [searchTerms['created'], searchTerms['changed'], searchTerms['creator'], searchTerms['changedby'], 
-      searchTerms['platform'], searchTerms['index_name'], searchTerms['index_seq'], searchTerms['box'], searchTerms['kitname']];
-    const detailedSampleTerms = [searchTerms['institute'], searchTerms['external']];
+    const plainSampleTerms = [searchTerms['created'], searchTerms['changed'], searchTerms['creator'], searchTerms['changedby'],
+        searchTerms['platform'], searchTerms['index_name'], searchTerms['index_seq'], searchTerms['box'], searchTerms['kitname']];
+    const detailedSampleTerms = [searchTerms['institute'], searchTerms['external'], searchTerms['groupid'], searchTerms['distributed'],
+        searchTerms['distributedto']];
```

Comment:
Distribution terms should be available in plain sample mode too. Same in list_samples.js